cmd/release-controller/release.go

```diff
@@ -335,3 +336,94 @@ func findTagReferencesByPhase(release *Release, phases ...string) []*imagev1.Tag
 	sort.Sort(tagReferencesByAge(tags))
 	return tags
 }
+
+func (c *Controller) findReleaseByName(includeStableTags bool, names ...string) (map[string]*ReleaseStreamTag, bool) {
```

Comment:
This should go into http_candidate until we have a reason to share it (even if that's 107)